### DISCUSSION TITLE
ABC East

### DISCUSSION THREAD

**Discussion by ghost** [iptv-org/community#1237] [Q&A]
Channel Name: ABC East
Country:US
Language:English
Stream URL (Optional:
Notes (Optional): I'm Searching for A Full HD Stream Of ABC East Stream URL More Then Just Local News

**Comment by ghost** [comment]
Channel Name: ABC East
Country:US
Language:English
Stream URL (Optional):http://my.streamerstv.com:25461/Bob123/Bob123/1675
Notes (Optional): I'm Searching for A Full HD Stream Of ABC East Stream URL More Then Just Local News

**Comment by ghost** [comment]
answers

**Comment by ghost** [comment]
correct

**Comment by ghost** [comment]
PLEASE

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/new?template=4_channel-search.yml
